packages/template-retail-react-app/app/components/toggle-card/index.jsx

```diff
@@ -34,6 +34,14 @@ export const ToggleCard = ({
     children,
     ...props
 }) => {
+    const titleRef = useRef()
+
+    useEffect(() => {
```

Comment:
Would you mind adding a unit test for this change?

Reply:
It has been done